### ISSUE TITLE
Will it be possible to self-host it?

### ISSUE THREAD

**Issue by agichim** [kochrt/timeline-maker#42]

**Comment by Semmu** [comment]
Yes please, it would be awesome to package this software as a Docker container!

**Comment by aaparmeggiani** [comment]
Not for production, but just submitted https://github.com/mark-when/markwhen/pull/108

**Comment by aaparmeggiani** [comment]
Moving the discussion to https://github.com/mark-when/markwhen/issues/71 , if that is ok (?)
